eppoclient/rules.go

```diff
@@ -39,27 +39,38 @@ func matchesRule(subjectAttributes dictionary, rule rule) bool {
 }
 
 func evaluateCondition(subjectAttributes dictionary, condition condition) bool {
-	subjectValue := subjectAttributes[condition.attribute]
+	subjectValue := subjectAttributes[condition.Attribute]
 
 	if subjectValue != nil {
-		if condition.operator == "MATCHES" {
+		if condition.Operator == "MATCHES" {
 			v := reflect.ValueOf(subjectValue)
 			if v.Kind() != reflect.String {
 				subjectValue = strconv.Itoa(subjectValue.(int))
 			}
-			r, _ := regexp.MatchString(condition.value.(string), subjectValue.(string))
+			r, _ := regexp.MatchString(condition.Value.(string), subjectValue.(string))
 			return r
-		} else if condition.operator == "ONE_OF" {
-			return isOneOf(subjectValue, condition.value.([]string))
-		} else if condition.operator == "NOT_ONE_OF" {
-			return isNotOneOf(subjectValue, condition.value.([]string))
+		} else if condition.Operator == "ONE_OF" {
+			return isOneOf(subjectValue, convertToStringArray(condition.Value))
+		} else if condition.Operator == "NOT_ONE_OF" {
+			return isNotOneOf(subjectValue, convertToStringArray(condition.Value))
 		} else {
 			return evaluateNumericCondition(subjectValue, condition)
 		}
 	}
 	return false
 }
 
+func convertToStringArray(conditionValue interface{}) []string {
+	if reflect.TypeOf(conditionValue).Elem().Kind() == reflect.String {
+		return conditionValue.([]string)
+	}
+	conditionValueStrings := make([]string, len(conditionValue.([]interface{})))
```

Comment:
Added this because Go does not allow converting `interface{}` to `[]string`. The unit test passed only because it used `[]string`. But the deserialized condition uses `interface{}`.